packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -49,6 +54,8 @@ export async function createSignSendProcessXCPDRequest({
     try {
       await executeWithNetworkRetries(async () => axios.post(pdResponseUrl, result));
     } catch (error) {
+      const errorString = errorToString(error);
+      log(errorString);
       capture.error("Failed to send PD response to Internal Carequality Endpoint", {
```

Comment:
Can we also send the message to the log? Just the error w/o context is not as helpful.

We have a "pattern" for this:

```ts
      const msg = "Failed to send PD response to Internal Carequality Endpoint";
      log(`${msg} - ${errorToString(error)}`);
      capture.error(msg, {
```